### ISSUE TITLE
Doesn't make profile.txt file.

### ISSUE THREAD

**Issue by EvilsSouls** [blackd/Inventory-Profiles#120]
The mod does not create the 'profiles-[world name].txt' file (which should be in .minecraft/config/inventoryprofilesnext/) when I open a world that I had created before downloading the mod (at least that's what I think it is why the mod isn't creating the file). I'm using 1.16.5 with the mod version 1.3.5. 
![image](https://user-images.githubusercontent.com/86242809/158697670-fc76b928-8f95-4126-8134-4398473de578.png)

**Comment by blackd** [comment]
**TLDR**. Above the button that opens the profiles file there is a button that opens the documentation about the profiles. Please fallow the instructions there on how to set them up.


That file is not created until the user attempts to create it by saving a state as a profile. That way we don't waste time wile the world is loaded/server connected. This benefit the vast majority of users which just use the sorting. Also setting the profiles up is quite involved and the missing file might be an early indication that the player should be directed to the documentation at  https://inventory-profiles-next.github.io/profiles/

If after you fallow the steps on how to create the file and your first profile you sitll get the error, it will best to contact us on discord so we can assist you.

**Comment by EvilsSouls** [comment]
Ohhh that makes sense, I looked at the instructions and just wanted to kinda... take a look at the file. I didn't think about that, thanks!
